src/Beman/Example/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Add the library.
+add_library(beman_example STATIC "")
+add_library(beman::example ALIAS beman_example)
+
+# Using the full project version because example does not provide a stable ABI
+# between shared library versions
+set_property(TARGET beman_example PROPERTY SOVERSION "${PROJECT_VERSION}")
+
+# Set the C++ sources.
+target_sources(beman_example PRIVATE
+  example.cpp
+)
+target_include_directories(
+  beman_example
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../../../include>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/${CMAKE_LOWER_PROJECT_NAME}> # <prefix>/include/scratch
+)
+# target_sources(example PUBLIC
+#   FILE_SET HEADERS
+#   BASE_DIRS ../../../
+#   FILES ../../../include/
+# )
```

Comment:
I think we should be using FILE_SET but here it is commented out. Would it also remove the need for the `include_directories` call?